### PR TITLE
graphene 1.10.2

### PR DESCRIPTION
**Pull request by ericLemanissier**
this is a requirement of gtk 4

Specify library name and version:  **graphene/1.10.2**

let's test it with FreeBSD

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/graphene/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        "with_glib": True,
+    }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
```
Comment (prince-chrismc) [inline recipes/graphene/all/conanfile.py:33]:
```suggestion
    
    def configure(self):
        if self.options.shared:
            del self.options.fPIC


```

File: recipes/graphene/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    
+    def build_requirements(self):
+        self.build_requires("meson/0.56.0")
+        self.build_requires("pkgconf/1.7.3")
+    
+    def requirements(self):
+        if self.options.with_glib:
+            self.requires("glib/2.67.0")
+
+    def configure(self):
```
Comment (prince-chrismc) [inline recipes/graphene/all/conanfile.py:42]:
```suggestion
    def configure(self):
        if self.options.shared:
            del self.options.fPIC
```

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/graphene/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
```
Comment (uilianries) [inline recipes/graphene/all/conanfile.py:4]:
```suggestion

required_conan_version = ">=1.29.0"
```

remove_files_by_mask is 1.29.0

**Review by uilianries — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Comment by ericLemanissier** [comment]
@SSE4 the only change since your review is the addition of `required_conan_version`. Would you have some time to approve this PR?

**Review by SSE4 — APPROVED** [review]

**Pull request merged**